components/brave_wallet_ui/common/slices/endpoints/token_balances.endpoints.ts

```diff
@@ -92,6 +97,9 @@ export type GetTokenBalancesRegistryArg = {
   accountIds: BraveWallet.AccountId[]
   networks: BalanceNetwork[]
   useAnkrBalancesFeature: boolean
+  /** if true, only spam NFT balances will be fetched, if falsey, only user
+   * token balances will be fetched */
+  isSpamRegistry?: boolean
```

Comment:
It makes sense to be consistent and comment with slashes `//`

Reply:
This was purposely commented using doc-comments to allow the comment to be visible when hovering over the variable within a code editor